### DISCUSSION TITLE
I can't import your dependency

### DISCUSSION THREAD

**Discussion by kaaaike** [patrickfav/bcrypt#64] [General]
Are we having any type of difficulty in this dependency? I cannot in any way import your lib into my project.

		<dependency>
			<groupId>at.favre.lib</groupId>
			<artifactId>bcrypt</artifactId>
			<version>0.10.2</version>
		</dependency>

Error:
Dependency 'at.favre.lib:bcrypt:0.10.2' not found

**Comment by patrickfav** [comment]
My guess is, you do not use Maven Central as your main Maven Repository. The dependency is clearly published: https://mvnrepository.com/artifact/at.favre.lib/bcrypt/0.10.2
